posthog/hogql_queries/experiments/experiment_trends_statistics.py

```diff
@@ -0,0 +1,134 @@
+from scipy import stats
+from posthog.schema import ExperimentSignificanceCode, ExperimentVariantTrendsBaseStats
+
+from ee.clickhouse.queries.experiments import (
+    FF_DISTRIBUTION_THRESHOLD,
+)
+
+Probability = float
+
+
+def posterior_poisson(total_counts, total_exposures, alpha, beta):
+    """Calculates the posterior distribution of a Poisson distribution given the data and prior parameters.
+
+    Parameters
+    ----------
+
+    total_counts: int
+      The total number of observed counts
+    total_exposures: int
+      The total number of exposed users
+    alpha: float
+      The prior hyper-parameters
+    beta: float
+      The prior hyper-parameter
+    """
+    prior = stats.gamma(a=alpha, scale=1 / beta)
+    alpha = alpha + total_counts
+    beta = beta + total_exposures
+    posterior = stats.gamma(a=alpha, scale=1 / beta)
+
+    return prior, posterior
+
+
+def calculate_probabilities(
+    control_variant: ExperimentVariantTrendsBaseStats,
+    test_variants: list[ExperimentVariantTrendsBaseStats],
+    simulations: int = 100_000,
```

Comment:
I guess this is an arbitrary large number for now, which is fine. we should do some quick testing to see if we really need such a large number